### PR TITLE
Tools

### PR DESCRIPTION
**Pull request by nullkarma**
Contains some handy actions like:
  - yaml reader
  - download
  - extract archive
  - ...

### CODE REVIEW

**Comment by nullkarma** [comment]
I'm not happy with this pack tbh. I'd prefer to add all the actions included in this pack as a part of core or st2 maybe. What's your opinion on that?
My intention was to have something written in python i can use to work with my gitlab workflows. Downloading and unzipping artifacts, reading content etc. - without fiddling around with core.(local|remote) stdout.

**Comment by LindsayHill** [comment]
I've been wondering about that too, which is partly why I didn't review this too closely yet. Haven't thought it all through, but `core` pack might well be the right home for at least some of the actions

File: stackstorm-tools/actions/yaml_reader.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env python
+
+from st2actions.runners.pythonrunner import Action
```
Comment (Kami) [inline stackstorm-tools/actions/yaml_reader.py]: We already have `csv` and `xml` pack so maybe this action should go into `yaml` pack :)

Comment (nullkarma) [inline stackstorm-tools/actions/yaml_reader.py]: Good point. Thanks Kami

File: stackstorm-tools/actions/relay.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+---
+
+name: relay
```
Comment (Kami) [inline stackstorm-tools/actions/relay.yaml]: Yeah, this one should probably just go into `core` or `mistral` pack

**Pull request closed without merging**